.gitignore

```diff
@@ -1,2 +1,3 @@
-www/script.js
 node_modules/
+www/script.js
+/venv
```

Comment:
Does this actually include everything in venv? That is quite interesting if so, because the git docs don't seem to indicate that would happen.

Reply:
I am pretty sure that this is standard in our Makefiles? What do you expect it to do?

Reply:
I expect it to just ignore a file named venv, but it appears to ignore folders recursively as well, interesting!